document-site.html

```diff
@@ -73,6 +73,10 @@ <h2>Components</h2>
                     <li>
                         Snackbar</li>
                 </a>
+                <a href="#rating" onclick="hideSideNavBar()">
```

Comment:
Interesting implementation, liked this logic and will make note of it for later use